training/model_management/components/import_model/spec.yaml

```diff
@@ -0,0 +1,135 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
+type: pipeline
+
+name: import_model
+display_name: Import model
+description: Import a model into a workspace or a registry
+version: 0.0.1
+
+# Pipeline inputs
+inputs:
+  # pipeline specific compute
+  compute:
+    type: string
+    description: Compute to run pipeline job
+
+  ## Inputs for download model
+  model_source:
+    type: string
+    description: Storage containers from where model will be sourced from.
+    default: Huggingface
+    enum:
+      - AzureBlob
+      - GIT
+      - Huggingface
+
+  model_id:
+    type: string
+    description: A valid model id for the model source selected. For example you can specify `bert-base-uncased` for importing HuggingFace bert base uncased model. Please specify the complete URL if **GIT** or **AzureBlob** is selected in `model_source`
+
+  ## Inputs for the MlFLow conversion
+  license_file_path:
```

Comment:
can we get rid of license file as input and have license file download as part of download component?

cc: @skasturi @HrishikeshGeedMS @ManojBableshwar

Reply:
How will it work for models that do not have the license file?

Reply:
I like the idea actually of making license file an output of the download component & make it an input to register component.

Reply:
We can take this up in next iterations.